comp110-worksheet-7/DirectoryUtils.cs

```diff
@@ -5,12 +5,60 @@
 using System.Text;
 using System.Threading.Tasks;
 
+/// <summary>
+/// Stores all of the data that is relevant to directory of this
+/// task, but can be used elsewhere provided it is given a root folder
+/// in the constructor.
+/// </summary>
+public class DirectoryData
```

Comment:
It's generally good practice to define each class in its own .cs file -- in fact Unity enforces this, so it's a good habit to get into.

Also, this class should really be in the same namespace as the `DirectoryUtils` class